Woof/Woof/Model/Owner/Owner+CurrentOwner.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+extension Owner {
+    /**
+     Handles the operations with current owner.
```

Comment:
I don't agree that we should specify any type of operation that can be called because any change of implementation causes the necessity to update the documentation.